### ISSUE TITLE
Make mega menu respect accessibility

### ISSUE THREAD

**Issue by jansensan** [jansensan/arts-et-medias#58]
Either mega menu should be available by keyboard navigation, or it should be skipped by keyboard nav.

https://adobe-accessibility.github.io/Accessible-Mega-Menu/
https://blogs.adobe.com/accessibility/2013/05/adobe-com.html
https://github.com/AllThingsSmitty/accessible-mega-menu
https://www.levelaccess.com/challenges-mega-menus-standard-menus-make-accessible/
https://soap.stanford.edu/tips-and-tools/tips/menus-navigation
http://www.terrillthompson.com/tests/menus/accessible-mega-menu/test.html
